#Exception Messages
user.not.found=Requested user with email {0} does not exist.
user.duplicate=User with email {0} already exists.
user.exception=Error while processing user request.
category.not.found=Requested category with id {0} does not exist.
category.duplicate=Category with name {0} already exists.
category.exception=Error while processing category request.
scope.not.found=Requested scope with id {0} does not exist.
scope.duplicate=Scope with name {0} already exists.
scope.exception=Error while processing scope request.